src/node/desktop/forge.config.js

```diff
@@ -65,32 +25,14 @@ const config = {
       config: {},
     },
     {
-      name: '@electron-forge/maker-dmg',
-      config: {
-        format: 'ULFO',
-        name: 'Rstudio-electron-app',
-        background: './resources/background/dmg-background.tiff',
-        icon: './resources/icons/RStudio.icns',
-        iconSize,
-        additionalDMGOptions: {
-          window: {
-            size: dmgSize,
-          },
-        },
-        contents: [
-          {
-            ...dmgIconPosition,
-            type: 'file',
-            path: __dirname + '/out/RStudio-darwin-x64/RStudio.app',
-          },
-          {
-            ...dmgApplicationsPosition,
-            type: 'link',
-            path: '/Applications',
-          },
-        ],
-      },
-    },
+      // we don't actually use the created zipfile; this is just a hack
+      // to make sure electron forge does the other packaging steps we
+      // want it to do without complaining we don't have any packaging
+      // steps defined
+      name: '@electron-forge/maker-zip',
```

Comment:
I _think_ our builds could switch to just running "electron-forge package" instead of "make" and eliminate this section altogether (thus skipping creation of the unused zipfile).

Conceptually "package" is put the electron bits in the standard layout for a given platform, including minifying the Javascript and all that fun.

"make" is "create a redistributable like a DMG or setup program from that created package" (which we are now doing outside of electron-forge so don't need to bother with)

These phases are described here:

https://www.electronforge.io/cli

Up to you if you want to try this as part of this PR, or do that separately.

Reply:
Aha! Thanks; I'll definitely do that as part of this PR. And thanks for the pointer to the documentation.